classes/controller/FrontController.php

```diff
@@ -494,6 +494,7 @@ public function init()
      */
     public function postProcess()
     {
+        Hook::exec('actionCaptchaValidation');
```

Comment:
I'm not a hook expert in PrestaShop but I don't  know if it's the good place to launch your hook here.

Reply:
I was also wondering if it was ok, I saw multiple modules using the postProcess overrides. If someone else could comment on this it would be great.